src/index.js

```diff
@@ -14,6 +14,11 @@ form.addEventListener('submit', (event) => {
   }
 });
 
+const btnClearAll = document.getElementById('btn-clear-task');
+btnClearAll.addEventListener('click', () => {
+  listTask.clearAll();
+});
```

Comment:
- [ ] Kindly consider adding the feature of deleting all the completed tasks when the user clicks the `clear all completed` button as specified in the project requirements

![image](https://user-images.githubusercontent.com/97100091/206214107-9b3c5a74-84ca-4961-8b8a-42344c5d9105.png)

Reply:
Hi, sorry, I forgot to update the Live demo. It's works now